torchrl/envs/transforms/transforms.py

```diff
@@ -3064,7 +3066,7 @@ def transform_input_spec(self, input_spec: CompositeSpec) -> CompositeSpec:
                 f"input_spec was expected to be of type CompositeSpec. Got {type(input_spec)} instead."
             )
         input_spec["step_count"] = UnboundedDiscreteTensorSpec(
-            shape=input_spec.shape,
+            shape=self.parent.done_spec.shape,
```

Comment:
In the tests, there is not parent env so 

`AttributeError: 'NoneType' object has no attribute 'done_spec'`

Reply:
I think we should default to the tensordict shape if there is not parent env.